qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/AdministrationManageRolesPage.java

```diff
@@ -15,7 +15,7 @@
 public class AdministrationManageRolesPage extends Page{
 
 	protected static final String PAGE_URL = "/openmrs/admin/users/role.list";
-	private static final By ADD_NEW_ROLE = By.cssSelector("#content > a");
+	private static final By ADD_NEW_ROLE_LINK= By.cssSelector("#content > a");
 	private static final By DELETE_SELECTED_ROLES = By.cssSelector("#content > form > input[type=submit]");
```

Comment:
is `DELETE_SELECTED_ROLES` a button, link or what?

Reply:
button